### DISCUSSION TITLE
Full custom Dashboard

### DISCUSSION THREAD

**Discussion by R1NEJ0** [filamentphp/filament#2361] [Ideas]
Maybe it's going to be good make Dashboard as a Page to customize it 100%.

**Comment by ryangjchandler** [comment]
@R1NEJ0 You can remove the default Dashboard by removing the class name in the config file and then create your own page. The default dashboard is just a page anyway.

**Reply by R1NEJ0** [reply]
I did it like you say but I wasn't pretty sure if it was the right way.

**Reply by danharrin** [reply]
Yeah it is, there's a reason why we expose the Dashboard as a pre-registered page like that.
